lute/cli/commands/setup/init.luau

```diff
@@ -50,11 +36,12 @@ if not table.find(args, "--with-luaurc") then
 end
 
 local luauRcPath = `{cwd}/.luaurc`
+local luauRcPathPathlike = path.parse(luauRcPath)
```

Comment:
do paths not have an `__tostring`? if they do (which they should), we can just parse it for the path instead of having two locals here